scripts/generate_protobuf.sh

```diff
@@ -39,7 +39,7 @@ rm -rf "$GRPC_GATEWAY_OUT_DIR"
 mkdir -p "$GRPC_GATEWAY_OUT_DIR"
 
 # go install github.com/grpc-ecosystem/protoc-gen-grpc-gateway-ts@latest
-PLUGIN_PATH_GRPC_GATEWAY="$(which protoc-gen-grpc-gateway-ts)"
+PLUGIN_PATH_GRPC_GATEWAY="/home/bob/go/bin/protoc-gen-grpc-gateway-ts"
```

Comment:
Why?

Reply:
Why you gotta be commenting on outdated code, dawg?

It's because your which doesn't work if go doesn't install that shit somewhere that's in PATH and I initially forgot to switch it back